rest/urls.py

```diff
@@ -39,4 +44,35 @@
         r'vocabularies/(?P<vocab_slug>[-\w]+)/terms/(?P<term_slug>[-\w]+)/$',
         TermDetail.as_view(),
         name='term-detail'),
+    # Section for repository group members
+    url(
+        r'^repositories/(?P<repo_slug>[-\w]+)/'
```

Comment:
All five of these URLs have ` r'^repositories/(?P<repo_slug>[-\w]+)/members' in common. Maybe it should be a constant above, making that more obvious and possibly shortening the regexes to fit on one line.